ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+namespace Homework
+{
+    public class Homework
+    {
+        static void Main(string[] args)
+        {
+            
+        }
+
+        //створити метод Max приймає від 2 до 5 аргументів та повертає max відповідно значення серед цих аргументів
+        public static int MaxV(int? v1, int? v2, int? v3 = null, int? v4 = null, int? v5 = null)
```

Comment:
it's ok, but do not use nullable until we're going to learn it

Reply:
Fixed